modules/tag/html/tag.js

```diff
@@ -0,0 +1,57 @@
+const nest = require('depnest')
+const { h, computed } = require('mutant')
+const hexrgb = require('hex-rgb')
+
+exports.gives = nest('tag.html.tag')
+
+exports.needs = nest({
+  'about.obs.color': 'first'
+})
+
+exports.create = function(api) {
+  return nest({ 'tag.html.tag': function({ tagName, tagId }, handleRemove) {
+    var removeTag
+    if (handleRemove) {
+      removeTag = h('a', {
+        'ev-click': handleRemove
+      }, 'x')
+    } else {
+      removeTag = '';
+    }
+
+    const backgroundColor = api.about.obs.color(tagId)
+    const fontColor = computed(backgroundColor, contrast)
+
+    return h(
+      'Tag',
+      {
+        style: {
+          'background-color': backgroundColor,
+          'color': fontColor
+        }
+      },
+      [
+        h('span', tagName),
+        removeTag
+      ]
+    )
+  }})
```

Comment:
:question: - your code looks good overall, I find your code quite spread out which makes it hard to read much without scrolling. e.g. the above is using pretty much as many lines as it could. Here's the same thing in significantly less lines and still quite readable IMO

```js
const style = {
  'background-color': backgroundColor,
  'color': fontColor
}
return h('Tag', { style }, [
  h('span', tagName),
  removeTag
])

```